.github/ISSUE_TEMPLATE/feature_request.yaml

```
@@ -0,0 +1,49 @@
+name: "🆕 Feature request"
+description: "Suggest an idea for this project"
+title: "🆕 Feature request: "
+labels: [feature]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Before opening a new issue, please search for duplicate issues to prevent opening a duplicate feature request.
+        If there is already an open existing request, please leave a comment there.
+
+        ## The Feature
+  - type: checkboxes
+    id: prerequisites
+    attributes:
+      label: "Prerequisites"
+      options:
+        - label: "I have searched for similar features requests in both open and closed tickets and cannot find a duplicate."
+          required: true
+        - label: "The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )"
+          required: true
+  - type: textarea
+    id: the_problem
+    validations:
+      required: true
+    attributes:
+      label: "What problem is your feature request going to solve? Please describe."
+      description: "A clear and concise description of what the problem is."
+      placeholder: "I'm always frustrated when ..."
```

```suggestion
      placeholder: "I'm always frustrated when..."
```

Also other places (I think... correct me if I'm wrong)

---

Also, regarding this, check the reference repo in the task description.